PHPCR/MediaManager.php

```diff
@@ -17,10 +17,6 @@ class MediaManager extends BasePHPCRManager
 {
     /**
      * {@inheritdoc}
-     *
-     * Warning: previous method signature was : save(MediaInterface $media, $context = null, $providerName = null)
```

Comment:
Wow, thas was here a long time before I contribute to sonata-project. :-)

ping @rande instead.